roles/tools/tasks/centos_9.yaml

```diff
@@ -1,46 +1,52 @@
 ---
-- name: Install extra tools
-  ansible.builtin.package:
-    name: "{{ item }}"
-    state: present
-  with_items:
-    - "{{ tools_packages }}"
-  become: yes
-  tags:
-    - sudo
-
-- name: Ensure user bin directory exists
-  ansible.builtin.file:
-    path: "{{ tools_user_bin_dir }}"
-    state: directory
-
-- name: Install Percona Toolkit dependencies
-  ansible.builtin.package:
-    name: "{{ tools_percona_toolkit_dependencies }}"
-    state: present
-  become: yes
-
-- name: Download Percona Toolkit tarball
-  ansible.builtin.get_url:
-    url: "{{ tools_percona_toolkit_url }}"
-    dest: "{{ tools_percona_toolkit_tmp_file }}"
-    checksum: "sha256:{{ tools_percona_toolkit_sha256 }}"
-
-- name: Extract Percona Toolkit tarball
-  ansible.builtin.unarchive:
-    src: "{{ tools_percona_toolkit_tmp_file }}"
-    dest: "/tmp"
-    remote_src: yes
-
-- name: Move Percona Toolkit files to user bin directory
-  shell: "mv {{ tools_percona_toolkit_tmp_dir }}/bin/* {{ tools_user_bin_dir }}"
-
-- name: Percona Toolkit install cleanup
-  ansible.builtin.file:
-    path: "{{ item }}"
-    state: absent
-  with_items:
-    - "{{ tools_percona_toolkit_tmp_file }}"
-    - "{{ tools_percona_toolkit_tmp_dir }}"
-
-...
\ No newline at end of file
+- name: Check for the percona toolkit tarball
+  ansible.builtin.stat:
+    path: "{{ tools_percona_toolkit_tmp_file }}"
```

Comment:
@aaronplagge this is stored in `/tmp` which means that this operation will trigger any time that systemd-tmpfiles has done a cleanup etc. It should be stored somewhere that is not prone to regular cleanup. That said, you should also be aware of [PT-2076](https://jira.percona.com/browse/PT-2076) :)

Reply:
With the release of the 3.5 pt packages I have adjusted the CentOS/RHEL 9 and Ubuntu 22 to use that instead.  Also fixed a couple of the ansible.builtin.packages to remove the with_items clause.